pkg/proto/runtime/v1/dapr_tracing.go

```diff
@@ -148,6 +148,15 @@ func (*StartWorkflowRequest) AppendSpanAttributes(rpcMethod string, m map[string
 func (*SubscribeConfigurationRequest) AppendSpanAttributes(rpcMethod string, m map[string]string) {
 	// TODO
 }
+func (*SubscribeTopicEventsInitialRequest) AppendSpanAttributes(rpcMethod string, m map[string]string) {
+	// TODO
```

Comment:
Is this to be added in a future PR?

Reply:
I  copied this from the other funcs in the [file](https://github.com/dapr/dapr/blob/a6bafa6babec9856259c6a1bb2a73beab0d245e8/pkg/proto/runtime/v1/dapr_tracing.go). Not entirely sure why they are `TODO`, but are needed to satisfy an interface.